Strain_Tools/strain/strain_tensor_toolbox.py

```diff
@@ -6,6 +6,21 @@
 import math as m
 
 
+def strain(dx,dy,V1,V2):
+    '''Compute strain rate on a regular grid'''
+    e11, dV1dx2 = np.gradient(V1, dx, dy)
```

Comment:
empirical discovery while debugging gpsgridder: the gradients are returned from this function in order of y-dimension followed by x-dimension.  I had assumed V1 was the x direction.  So... I'm inclined to switch the order of the two return variables.  Please advise- will doing so break your usage of this function?

Reply:
@kmaterna I'm not sure, haven't got far enough to check the strain calculation yet, only verified that the velocity calculation was working. You can check my "makeGrid" function; I think it should be x/y, but now there could be an issue with long/lat vs lat/lon interfacing with x/y. I'll check later tonight and let you know

Reply:
In general I would say just do what makes gpsgridder work correctly for the moment, and I can adjust mine if needed.

Reply:
sounds good!